README.md

```diff
@@ -122,13 +122,61 @@ Configuration
 -------------
 
 When pushing the controller into a Cloud Foundry environment a configuration must be passed
-via the application manifest. An example manifest is located in the example_config directory.
+via the application manifest. An example manifest is located in the
+`docs/_static/config_examples` directory.
 
 Update required sections for environment:
 - nats: leave empty for gnatsd otherwise update with CF installed NATS information
 - bigip: leave empty if no BigIP is required otherwise update with BigIP information
-- routing_api: only required if routing API access is required
-- oauth: only required if routing API access is required
+- routing_api: only required if routing API access is required (TCP routing with route_mode
+set to all, or tcp)
+- oauth: only required if routing API access is required (TCP routing with route_mode
+set to all, or tcp)
+
+On startup, the controller will get the `BIGIP_CTLR_CFG` variable from the environment;
+setting this variable via an application manifest is required, environment parameters
+are set in the manifest `env` section. The controller also supports cloud foundry health
```

Comment:
cloud foundry should be capitalized - couple of instances need updated

Reply:
Fixed